docs/how-to-guides/integrating-autoware/creating-your-autoware-meta-repository/creating-autoware-meta-repository.md

```diff
@@ -0,0 +1,57 @@
+# Creating Autoware meta-repository
+
+## What is Meta-repository?
+
+A meta-repository is a repository that manages multiple repositories, and Autoware is one of them.
```

Comment:
`Autoware` which is system name is not a meta-repository , but [autoware](https://github.com/autowarefoundation/autoware) which is repository  is a meta-repository.

How about using links to express this?

Reply:
fix https://github.com/autowarefoundation/autoware-documentation/pull/414/commits/06a5739468105b4fa14062901a9543758a579f3f